graylog2-web-interface/src/theme/constants.ts

```diff
@@ -20,11 +20,12 @@ export type ThemeMode = 'teint' | 'noir';
 export type PreferencesThemeMode = 'themeMode';
 
 const PREFERENCES_THEME_MODE: PreferencesThemeMode = 'themeMode';
-const ROOT_FONT_SIZE = 14;
+const ROOT_FONT_SIZE = 14; // This value is also being maintained as @font-size-base in bootstrap-config.js.
```

Comment:
Why @font-size-base and  this one are different? Is it should be like that?

Reply:
Currently it is the equivalent to body font size.
https://github.com/Graylog2/graylog2-server/pull/13425/files#diff-ea1ddcace101558dd56650177440aef11fa139013d8183c04f8e26ec1b02e291R76

But I will remove the comment, I realised it does not make sense anymore.

Reply:
Unfortunately the value can not be defined in `rem`.